catroid/src/main/java/org/catrobat/catroid/common/Constants.java

```diff
@@ -39,7 +39,7 @@
 
 public final class Constants {
 
-	public static final double CURRENT_CATROBAT_LANGUAGE_VERSION = 0.99999999;
+	public static final double CURRENT_CATROBAT_LANGUAGE_VERSION = 0.999999991;
```

Comment:
```suggestion
	public static final double CURRENT_CATROBAT_LANGUAGE_VERSION = 0.999999992;
```
In the meantime other xml changes have already happened:
https://github.com/Catrobat/Catroid/blob/develop/catroid/src/main/java/org/catrobat/catroid/common/Constants.java#L42